sql/rpl_slave.cc

```diff
@@ -725,6 +725,20 @@ bool start_slave_cmd(THD *thd) {
     }
 
     res = start_slave(thd);
+#ifdef WITH_WSREP
+  // Similar to GR, do not allow operations on the wsrep channel
+  } else if (lex->mi.channel && wsrep_is_wsrep_channel_name(lex->mi.channel)) {
```

Comment:
I am not sure if these syntax are applicable to wsrep directly.
For async slave, user is allowed to start slave thread and stop it. (GR may follow the same).
In case of wsrep it is not allowed. So I don't foresee a use-case for this but if u you find one please help add it here along with test-case.

Reply:
I think the issue is that someone could create an async channel using the wsrep filter.